<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IROOTLoopModule</class>
 <widget class="QWidget" name="IROOTLoopModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 105, 73);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="title">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Loop Settings (use %1 as placeholder)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="close">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/metro.icons/black/cancel.png</normaloff>:/icons/icons/metro.icons/black/cancel.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="loopoverIndex">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Loop over index</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="loopoverFiles">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Loop over files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="loopoverFilesandIndex">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Loop over files and then index</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="loopoverIndexandFiles">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Loop over index and then files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Index begin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="loopBegin">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="loopEnd">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Increment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="loopIncrement">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pad Changeover</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="padChangeOver">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>loopoverIndex</zorder>
  <zorder>loopoverFiles</zorder>
  <zorder>loopoverFilesandIndex</zorder>
  <zorder>loopoverIndexandFiles</zorder>
  <zorder>line</zorder>
  <zorder>titleWidget</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
